src/runtime/link-accounts-flow.js

```diff
@@ -219,15 +215,43 @@ export class LinkSaveFlow {
    * @return {!Promise}
    */
   start() {
+    const iframeArgs = {
+      'publicationId': this.deps_.pageConfig().getPublicationId(),
+      'isClosable': true,
+    };
+    /** string */
+    const token = (() => {
```

Comment:
Hmm. This is slightly confusing to me. Why is this block closurized and resolved by function invocation? Also, at this time the `request_` is defined as a non-null value, why check for its existence below? Finally, the request is a typedef, so you should access it's properties directly w/o quotes. 

In other words, isn't this just:
```
const token = this.request_.token;
```

Reply:
Done.